roboto/request_types.py

```diff
@@ -296,3 +296,14 @@ class StopPollRequest:
     chat_id: Union[ChatID, str]
     message_id: MessageID
     reply_markup: Optional[JSONSerialized[InlineKeyboardMarkup]] = None
+
+
+@dataclass(frozen=True)
+class DiceRequest:
+    """Parameters for requesting a Dice."""
+
+    chat_id: Union[ChatID, str]
+    emoji: Optional[str] = ':dice:'
```

Comment:
`= 'None'`. We don't mirror the API defaults because they may one day change (Although that would be a breaking change. But we avoid doubling the work.)